langkit/expressions/base.py

```diff
@@ -1690,6 +1714,9 @@ def render_public_ada_constant(self):
     def render_python_constant(self):
         return repr(self.value)
 
+    def render_ocaml_constant(self):
+        return '(Char.code {})'.format(ord(self.value))
```

Comment:
Curious about why you need this ? Should probably document why code equivalent to the python one isn't sufficient

Reply:
Actually, this is pretty tricky, and is false for now, good catch.
I guess that characters can be unicode?
In OCaml bindings, characters are represented as utf-8 strings.
So I guess that I should also make the convertion here

Reply:
Yes, in Langkit, strings are represented as arrays of wide wide characters (i.e. full Unicode codepoints). If the idiomatic type to use in OCaml is UTF-8 strings, yes: do the conversion each time a value goes from our C API through the OCaml binding and conversely.